dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+include(FetchContent)
+
+FetchContent_Declare(sqlpp1
+    GIT_REPOSITORY  https://github.com/rbock/sqlpp11
+    GIT_TAG         origin/develop
+)
+
+FetchContent_Declare(date
+    GIT_REPOSITORY https://github.com/HowardHinnant/date
+    GIT_TAG        v3.0.0
+)
+
+FetchContent_GetProperties(sqlpp1)
+if (NOT sqlpp1_POPULATED)
+    FetchContent_Populate(sqlpp1)
+    add_subdirectory(${sqlpp1_SOURCE_DIR} ${sqlpp1_BINARY_DIR})
+endif()
```

Comment:
As in the other PR, I would suggest to use FetchContent_MakeAvailable here. I still don't get any error when using it here. So perhaps we could change this to MakeAvailable for testing purposes and you could make https://github.com/GCarneiroA/sqlpp11_mysql_test fail?
Then the error should be reproducable for me, right? I can then check then what's going on :)